django/db/backends/sqlite3/features.py

```diff
@@ -24,7 +25,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     can_rollback_ddl = True
     can_create_inline_fk = False
     supports_paramstyle_pyformat = False
-    can_clone_databases = True
+    can_clone_databases = PY37
```

Comment:
Django 4.0+ will not support Python 3.7 anymore:
```suggestion
    can_clone_databases = True
```